tests/page-objects/contacts/contacts.po.js

```diff
@@ -125,4 +127,13 @@ module.exports = {
 
   getReportsFilters: () => element.all(by.css('.card.reports .table-filter a')),
   getTasksFilters: () => element.all(by.css('.card.tasks .table-filter a')),
+
+  waitForActionbarButtons: (nonAdminUser) => {
```

Comment:
I wonder if this would be better used in the `common.po.js` and added to the end of the `goToX` so that all tests benefit from this. It already seems like it's reuse is beneficial based on the changes to `selectFormNative` below.  I'm just not sure if it would cause issues with other tests. if we made a sweeping change.

Reply:
This is what I did 1st and then changed my mind :D

Reply:
In this commit: https://github.com/medic/cht-core/pull/6970/commits/0116dc5a82192e4a9336282dd5d492de8347c9a0
I actually had one in common and moved it to it's "rightful" place (in contacts). 

The reason was: when we add offline users ... it's going to be weird to micromanage every "goToThing" and pass an extra parameter. So I've put them where they were actually needed.

Reply:
That makes sense to me. 👍